### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Updated the main README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,98 @@
...
+
+# WARNING: Alpha release
+
+KubeLinter is at an early stage of development. There may be breaking changes in
+the future to the command usage, flags, and configuration file formats. However,
+we encourage you to use KubeLinter to test your environment YAML files, see what
+breaks, and [contribute](./CONTRIBUTING.md).
 
-# WARNING: Breaking changes possible
+# LICENCE
```
Comment (gaurav-nelson) [inline README.md:96]:
```suggestion
# LICENSE 
```

File: README.md [hunk 2]
```diff
@@ -1,33 +1,98 @@
...
+KubeLinter is a static analysis tool for identifying problematic patterns in
+Kubernetes configuration files and Helm charts. It enables you to make sure that
+the applications represented by those files adhere to best practices.
 
-In detail, `kube-linter` is a binary that takes in paths to YAML files, and runs a list of checks
-against them. If any lint errors are found, they are printed to standard error, and `kube-linter` returns a non-zero 
-exit code.
+The KubeLinter accepts YAML files as input and runs a series of checks on them.
+When it finds any issues, it reports those issues and returns a non-zero exit
+code.
```
Comment (viswajithiii) [inline README.md:9]:
```suggestion
KubeLinter accepts YAML files as input and runs a series of checks on them.
If it finds any issues, it reports them and returns a non-zero exit
code.
```

File: README.md [hunk 3]
```diff
@@ -1,33 +1,98 @@
 # KubeLinter
 
-KubeLinter is a static analysis tool that checks Kubernetes YAML files and Helm charts
-to ensure the applications represented in them adhere to best practices.
+KubeLinter is a static analysis tool for identifying problematic patterns in
+Kubernetes configuration files and Helm charts. It enables you to make sure that
+the applications represented by those files adhere to best practices.
```
Comment (viswajithiii) [inline README.md:5]:
```suggestion
KubeLinter is a static analysis tool that checks Kubernetes YAML files and Helm charts	KubeLinter is a static analysis tool for identifying problematic patterns in
to ensure the applications represented in them adhere to best practices.
```

File: README.md [hunk 4]
```diff
@@ -1,33 +1,98 @@
...
-against them. If any lint errors are found, they are printed to standard error, and `kube-linter` returns a non-zero 
-exit code.
+The KubeLinter accepts YAML files as input and runs a series of checks on them.
+When it finds any issues, it reports those issues and returns a non-zero exit
+code.
 
-The list of checks that is run is configurable. `kube-linter` comes with several built-in checks, only some of which
-are enabled by default. Users can also create custom checks.
+KubeLinter is:
+- **Configurable**: include multiple built-in checks which you can enable or
```
Comment (viswajithiii) [inline README.md:12]:
```suggestion
- **Configurable**: includes multiple built-in checks which you can enable or
```

File: README.md [hunk 5]
```diff
@@ -1,33 +1,98 @@
...
+  disable.
+- **Extensible**: you can define and configure custom templates and checks.
 
-## Install
+## Installation
 
-If you have `go` installed, you can run `go get golang.stackrox.io/kube-linter/cmd/kube-linter`.
+If you have installed [Go](https://golang.org/), run the following command:
```
Comment (viswajithiii) [inline README.md:18]:
```suggestion
If you have [Go](https://golang.org/) installed, run the following command:
```

File: README.md [hunk 6]
```diff
@@ -1,33 +1,98 @@
...
+> Go](https://golang.org/doc/install).
+
+To build KubeLinter from source, follow these instructions:
+
+1. Clone the KubeLinter repository.
+   ```bash
+   git clone git@github.com:stackrox/kube-linter.git
+   ```
+1. Run the `make build` command. This command compiles the source code and
+   creates a `kube-linter` binary file in the `kube-linter/bin/` folder.
```
Comment (viswajithiii) [inline README.md:39]:
```suggestion
   creates `kube-linter` binary files for each platform in the `.gobin` folder.
```

File: README.md [hunk 7]
```diff
@@ -1,33 +1,98 @@
...
+we encourage you to use KubeLinter to test your environment YAML files, see what
+breaks, and [contribute](./CONTRIBUTING.md).
 
-# WARNING: Breaking changes possible
+# LICENCE 
 
-kube-linter is currently in a very early stage of development. There may be breaking changes to the command usage, flags
-and config file formats.
\ No newline at end of file
+KubeLinter is licensed under [Apache License 2.0](./LICENSE).
```
Comment (viswajithiii) [inline README.md:98]:
```suggestion
KubeLinter is licensed under the [Apache License 2.0](./LICENSE).
```

File: README.md [hunk 8]
```diff
@@ -1,33 +1,98 @@
...
+       - name: sec-ctx-vol
+         mountPath: /data/demo
+       securityContext:
+         allowPrivilegeEscalation: false
+   ```
+1. To lint this file with KubeLinter, run the follwoing command:
+   ```bash
+   kube-linter lint pod.yaml
+   ```
+1. KubeLinter runs the built-in checks and report errors.
```
Comment (viswajithiii) [inline README.md:75]:
```suggestion
1. KubeLinter runs the default checks and reports errors.
```

**Review by viswajithiii — APPROVED** [review]
Awesome, thank you!

File: README.md [hunk 9]
```diff
@@ -1,33 +1,97 @@
...
+KubeLinter accepts YAML files as input and runs a series of checks on them.
+If it finds any issues, it reports them and returns a non-zero exit
+code.
 
-The list of checks that is run is configurable. `kube-linter` comes with several built-in checks, only some of which
-are enabled by default. Users can also create custom checks.
+KubeLinter is:
+- **Configurable**: includes multiple built-in checks which you can enable or
+  disable.
+- **Extensible**: you can define and configure custom templates and checks.
```
Comment (viswajithiii) [inline README.md:13]:
```suggestion
- **Extensible**: you can define and configure custom checks.
```

**Pull request merged**